zcash_keys/src/keys.rs

```diff
@@ -628,39 +657,52 @@ impl UnifiedFullViewingKey {
     /// Returns `None` if the specified index does not produce a valid diversifier.
     pub fn address(
         &self,
-        j: DiversifierIndex,
-        request: UnifiedAddressRequest,
+        _j: DiversifierIndex,
+        _request: UnifiedAddressRequest,
     ) -> Result<UnifiedAddress, AddressGenerationError> {
         #[cfg(feature = "orchard")]
-        let orchard = if request._has_orchard {
```

Comment:
Hmm, IIRC this is a bugfix to how `UnifiedAddressRequest` works. We probably want to get this PR in before `zcash_keys 0.1.0` in that case, which means the earlier feature flag issue I raised in the PR needs to be fixed.